.github/workflows/continuous-integration-workflow.yml

```diff
@@ -450,6 +450,89 @@ jobs:
           ./mvnw -B clean package exec:java -Dexec.args="$GITHUB_WORKSPACE"
 
 
+  db2-install:
+    name: Test DB2 install script
+    needs: [build, should-do-database-tests, check_branch]
+    runs-on: ubuntu-latest
+    if: ${{ needs.should-do-database-tests.outputs.check == 'true' || needs.check_branch.outputs.is_publishable_branch == 'true'}}
+    steps:
+      - name: Checkout Openfire
+        uses: actions/checkout@v4
+      - name: Set up JDK 17 Zulu
+        uses: actions/setup-java@v4
+        with:
+          java-version: 17
+          distribution: zulu
+          cache: maven
+      - name: Restore mvn repo artifacts from build job
+        uses: actions/download-artifact@v4
+        with:
+          name: mvn-repo
+          path: ~/.m2/repository/org/igniterealtime/openfire/
+      - name: Set environment variables
+        run: |
+          echo "CONNECTION_STRING=jdbc:db2://localhost:50000/openfire" >> $GITHUB_ENV
+          echo "CONNECTION_DRIVER=com.ibm.db2.jcc.DB2Driver" >> $GITHUB_ENV
+          echo "CONNECTION_USERNAME=sa" >> $GITHUB_ENV
+          echo "CONNECTION_PASSWORD=SecurePa55w0rd" >> $GITHUB_ENV
+          OPENFIREVSN=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout)
+          echo "OPENFIREVSN=$OPENFIREVSN" >> $GITHUB_ENV
+          echo "JAVA_HOME=$(echo $JAVA_HOME_17_X64)" >> $GITHUB_ENV
+      - name: Start database server with an empty database
+        run: |
+          mkdir olddb
+          echo 'CREATE TABLE imready AS (SELECT 1);' > $GITHUB_WORKSPACE/olddb/ZZZ-signal-ready.sql
+          docker compose -f ./build/ci/compose/db2.yml up --detach
+      - name: Execute Openfire's database install script
+        run: |
+          pushd ./build/ci/updater
+          ./mvnw package
+          java -jar ./target/updaterunner-1.0.0-jar-with-dependencies.jar
```

Comment:
The -with-dependencies suffix is failing the build right now

Reply:
Ah, that was replaced by the Maven exec plugin for other databases. I forgot to change that for the DB2 branch, which was developed in parallel with the changes for other databases.

I've applied the same change to DB2 now.